packages/application-components/src/components/dialogs/info-dialog/info-dialog.story.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text, select, boolean } from '@storybook/addon-knobs';
+import withReadme from 'storybook-readme/with-readme';
+import { PORTALS_CONTAINER_ID } from '@commercetools-frontend/constants';
+import {
+  SecondaryButton,
+  Spacings,
+  Text,
+} from '@commercetools-frontend/ui-kit';
+import Readme from './README.md';
+import InfoDialog from './info-dialog';
+
+const DialogController = props => {
+  const [isOpen, toggle] = React.useState(false);
+  return (
+    <div
+      style={{
+        display: 'grid',
+        alignItems: 'center',
+        justifyContent: 'center',
+        height: '100%',
+        width: '100%',
+      }}
```

Comment:
Sorry, can we extract this? Either in a .mod.css or emotion. I don't feel like we have a rule for inline styles apart from: "it's not that much this time is it".

Reply:
We plan to migrate to emotion, so this is just temporary.

Reply:
I also don't mind using inline styles in stories.